packages/@roots/bud-api/src/assets.ts

```diff
@@ -42,7 +42,7 @@ export const assets: Assets = function (jobs) {
       }
 
```

Comment:
This entire file can be removed. It is now defined along with the `copy-webpack-plugin` extension in `@roots/bud`.